### PR TITLE
Add darktrace vsensor standalone template

### PR DESCRIPTION
**Pull request by menev-dt**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by menev-dt** [comment]
Hi,
   
May I ask someone, presumably Brian Moore, to take a look at this Pull request?

Thanks,
Martin

File: application-workloads/darktrace-vsensor-standalone/README.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+##### Access to the vSensor
+The vSensors created will not have public IP associated. Exposing a vSensor via a public IP carries additional security burden and is not a requirement of push token authentication with the master instance.
+
+It is expected that there is already a solution in place to access the VMs in the VNet via their private IP, for example via Azure Bastion Host.
+
+If there is a virtual firewall on your network, please ensure access is granted to the vSensor to the Darktrace instance FQDN/port or IP/port, as well as to packages.darktrace.com and packages-cdn.darktrace.com on port 443.
+
+
+
+How to deploy the template with Azure Portal
```
Comment (bmoore-msft) [inline application-workloads/darktrace-vsensor-standalone/README.md:138]: remove this section, all the user has to do is click "Deploy to Azure" button above (and it's the same for all templates in the repo)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/darktrace-vsensor-standalone/README.md (hunk 1 above)
Comment (menev-dt) [reply application-workloads/darktrace-vsensor-standalone/README.md]: the section was removed

**Comment by bmoore-msft** [comment]
Can you resolve the merge conflicts?

**Pull request merged**